manga-finder/src/pages/Error.tsx

```diff
@@ -0,0 +1,12 @@
+function Error() {
+  return (
+    <div className="error-page">
+      <div className="error-404 flex flex-col text-2xl">
+        <h2>Whoops! Error404</h2>
+        <img src="./src/assets/baka.jpeg" alt="baka!" />
```

Comment:
The image doesn't appear for routes like `/manga/{manga-id}/foo` or `/browse/foo`.